<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>568</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>641</width>
      <height>341</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <row>
     <property name="text">
      <string notr="true"/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="CrossPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="accessories-character-map">
       <normaloff>C:/Users/Sajad Ghandehari/.designer/backup</normaloff>C:/Users/Sajad Ghandehari/.designer/backup</iconset>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LastName</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>City</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ClasicTheme">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>540</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FBF3D5; /* Button Color */
    color: #FBF3D5; /* Text Color */
    border-radius: 10px; /* Half the button's height to make it circular */
    width: 20px; /* Adjust width and height as needed */
    height: 20px;
}

QPushButton:hover {
    background-color: #D8D8D8; /* Darker color on hover */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="darktheme">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #395B64; /* Button Color */
    color: #395B64; /* Text Color */
    border-radius: 10px; /* Half the button's height to make it circular */
    width: 20px; /* Adjust width and height as needed */
    height: 20px;
}

QPushButton:hover {
    background-color: #005f80; /* Darker color on hover */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>641</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="Name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>25</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit_button">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>27</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="City">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>25</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>25</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>City :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LastName">
     <property name="geometry">
      <rect>
       <x>263</x>
       <y>25</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>25</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>LastName :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionNew_file">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Sajad Ghandehari/.designer/icon/file2.png</normaloff>C:/Users/Sajad Ghandehari/.designer/icon/file2.png</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionfile_2">
   <property name="text">
    <string>file 2</string>
   </property>
  </action>
  <action name="actionwell_flow">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Sajad Ghandehari/.designer/icon/oilwell.png</normaloff>C:/Users/Sajad Ghandehari/.designer/icon/oilwell.png</iconset>
   </property>
   <property name="text">
    <string>well flow</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Sajad Ghandehari/.designer/icon/play.png</normaloff>C:/Users/Sajad Ghandehari/.designer/icon/play.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionopen_file">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Sajad Ghandehari/.designer/icon/openFile.png</normaloff>C:/Users/Sajad Ghandehari/.designer/icon/openFile.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Sajad Ghandehari/.designer/icon/savefile.png</normaloff>C:/Users/Sajad Ghandehari/.designer/icon/savefile.png</iconset>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
